docs/user/templates.rst

```diff
@@ -116,6 +122,9 @@ unassigned from a device configuration, they can only be overridden.
 They will be always assigned earlier than default templates, so they can
 be overridden if needed.
 
+Another difference to :ref:`default_templates` is that required templates are
+assigned to all devices, not just new devices.
```

Comment:
Are you 100% sure of this? I don't remember we added logic to automatically modify existing devices. As far as I remember the required template would be applied only if the user saves the device object.

Reply:
I didn't see it in the code but I noticed that a required template had been marked as assigned (with its checkbox disabled) in all devices after the template was made required. I didn't create a new device at all.